prov/efa/src/efa_cq.c

```diff
@@ -118,68 +136,79 @@ static void efa_cq_read_data_entry(struct efa_wc *wc, int i, void *buf)
 	entry[i].len = (uint64_t)wc->ibv_wc.byte_len;
 }
 
+/**
+ * @brief Adjust an error code from CQ poll API, e.g. `ibv_start_poll`, `ibv_end_poll`.
+ * The returned error code must be 0 (success) or negative (error).
+ * As a special case, if input error code is ENOENT (there was no item on CQ), we should return -FI_EAGAIN.
+ * @param[in] err	Return value from `ibv_start_poll` or `ibv_end_poll`
+ * @returns Adjusted error code
+ */
+static inline ssize_t efa_cq_poll_adjust_error(ssize_t err) {
```

Comment:
`efa_cq_poll_adjust_error` -> `efa_cq_ibv_poll_error_to_fi_error`

Reply:
I want to clarify this comment
> The unit test refactor commit should be merged into unit test

My understanding is to squash both commits into one. Business logic change should be accompanied by unit tests.

Do you mean different?

Reply:
Sounds good. I squashed both commits.